Contents/mods/CommunityAPI/media/lua/client/ModSettingAPI/ModSandboxSetting.lua

```diff
@@ -0,0 +1,130 @@
+local jsonUtils = require("CommunityAPI/JsonUtils")
+
+---@class ModSandboxSetting
+ModSandboxSetting = {}
```

Comment:
Usage of global must be as low as possible.
If possible change to local and return at the end of the file.
Require into a local variable where needed.